src/activity-functions-function-notation.xml

```diff
@@ -583,17 +485,22 @@
                 </set>
               </var>
               <pg-code>
-                $table1 = RadioButtons(["This table is a function","This table is not a function"],1);
-                $table2 = RadioButtons(["This table is a function","This table is not a function"],0);
+                $table1 = RadioButtons(["Table A is a function","Table A is not a function"],1);
+                $table2 = RadioButtons(["Table B is a function","Table B is not a function"],0);
               </pg-code>
             </setup>
             <statement>
-              <p>Is the table of values a function or not?</p>
-              <table>
+              <p>Is table A a function or not?</p>
+              <p><em>Table A</em></p>
```

Comment:
Here (within a WeBWorK) for sure `<title>` is not implemented. But I believe `<caption>` is. Same issue with Table B.